tests/Umbraco.Tests.AcceptanceTest/cypress/integration/Languages/languages.ts

```diff
@@ -1,10 +1,31 @@
 /// <reference types="Cypress" />
+
 context('Languages', () => {
 
     beforeEach(() => {
         cy.umbracoLogin(Cypress.env('username'), Cypress.env('password'), false);
       });
 
+    it('Creates language', () => {
+      // Setup
+      const language = 'Danish';
+      const culture = 'da';
+      cy.umbracoEnsureLanguageNotExists(culture);
```

Comment:
```suggestion
      cy.umbracoEnsureLanguageNameNotExists(culture);
```
Use the already existing language method